### DISCUSSION TITLE
Auth Email Verification Taking Really Long

### DISCUSSION THREAD

**Discussion by BMTCompany2** [supabase/supabase#11813] [Questions] [answered]
I'm using auth by email for users to sign up to my react app.  Following the standard workflow, users must verify there email, then follow a link to sign into the website.

Everything basically works with this.

The issue I'm facing is that its taking about 5min for the verification email to show up in users' inbox.  This is a poor user experience as users don't know if the process is working properly.  My understanding is that the verification email should show up near instantly.  Does anyone know where there would be a big lag?

Thanks!

**Comment by hf** [comment, marked as answer]
Hi, please read our [Production Readiness](https://supabase.com/docs/guides/platform/going-into-prod#availability) guide specifically the Availability part.

Email sending functionality on all tiers is best-effort only and meant to get you started. You should bring your own SMTP server.

**Reply by vipulb2** [reply]
A suggestion of e-mail providers is a better answer.👍
